core/src/main/kotlin/com/exactpro/th2/read/db/core/ValueTransformProvider.kt

```diff
@@ -16,46 +16,48 @@
 
 package com.exactpro.th2.read.db.core
 
-import com.exactpro.th2.read.db.core.impl.DefaultValueTransformer
+import com.exactpro.th2.read.db.core.impl.DefaultValueTransform
 import java.net.URI
+import java.sql.Connection
 import java.util.ServiceLoader.load
 
-typealias Transformer = (Any) -> Any
-typealias ToStringTransformer = (Any) -> String
+typealias ToNullableStringTransformer = (Any?, Connection) -> String?
```

Comment:
note: Please, add a comment about why we need the `Connection` object here. Because without explanation it looks like something wrong is going on here (and it sure does...). But as I understand some of the types in Oracle DB cannot be properly converted without having `Connection` object (it is used to get a time zone of DB). And I spent some time to figure this out. Let's leave some clues here